src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/SafeEvpPKeyHandle.OpenSsl.cs

```diff
@@ -11,6 +11,12 @@ public sealed partial class SafeEvpPKeyHandle : SafeHandle
     {
         internal static readonly SafeEvpPKeyHandle InvalidHandle = new SafeEvpPKeyHandle();
 
+        /// <summary>
+        /// In some cases like when a key is loaded from a provider, the key may have an associated data
+        /// we need to keep alive for the lifetime of the key. This field is used to track that data.
+        /// </summary>
+        internal IntPtr ExtraHandle { get; private set; } = IntPtr.Zero;
```

Comment:
Nit: The initializer assignment is redundant with the calloc of the object from `new`; it can be removed.

```suggestion
        internal IntPtr ExtraHandle { get; private set; };
```